Dockerfile.dapper

```diff
@@ -49,15 +47,19 @@ RUN curl -sLf ${!HELM_URL_V2} -o /usr/bin/rancher-helm && \
     chmod +x /usr/bin/rancher-helm /usr/bin/rancher-tiller && \
     ln -s /usr/bin/rancher-helm /usr/bin/helm && \
     ln -s /usr/bin/rancher-tiller /usr/bin/tiller && \
-    helm init -c --stable-repo-url https://charts.helm.sh/stable/ && \
-    helm plugin install https://github.com/rancher/helm-unittest
+    helm init -c --stable-repo-url https://charts.helm.sh/stable/
 
 # set up helm 3
 RUN mkdir /usr/tmp && \
     curl ${HELM_URL_V3} | tar xvzf - --strip-components=1 -C /usr/tmp/ && \
     mv /usr/tmp/helm /usr/bin/helm_v3 && \
     chmod +x /usr/bin/kustomize
 
+# set up helm unittest - doesn't need to run for s390x architecture
+RUN if [ "${ARCH}" == "arm64" || "${ARCH}" == "amd64" ]; then \
+        helm plugin install https://github.com/helm-unittest/helm-unittest.git --version 0.3.2 \
```

Comment:
This version still needs to be in an environment variable for easy (automated) updating